UI/CardUISetter.gd

```diff
@@ -5,12 +5,10 @@ class_name CardUISetter
 ## The idea with using a UI setter like this is we don't have any references to UI stuff
 ## in the actual Card class, so we can keep that class as minimal as possible.
 
-
-@export var title_label: Label
-@export var description_label: Label
-@export var energy_label: Label
 @export var key_art: TextureRect
-
+@export var attack_ui: Control
+@export var skill_ui: Control
+@export var power_ui: Control
 
```

Comment:
so you replace all with a specific UI for each type of card then ?

Reply:
Yes, Since the card art for Attack, Skill, and Power varies slightly it was pretty tough trying to just use one Control to set the text and have it be aligned properly. So I created three separate Control objects for each Card Type so I can set the text individually for each type. 

It does mean if we wanted to change the location of a text, I'd have to do it three times for each card type but seeing as how we're only gonna have three anyways I think we can handle that burden.

Reply:
If it's only three times, and it's better to fit everything like that, then that's a good thing